### ISSUE TITLE
Reset password

### ISSUE THREAD

**Issue by DominikRicko** [DominikRicko/AdministrationSystem#11]
- [x] Password reset process is initiated by the user pressing a "Reset password" button or hyperlink, which leads to a website where the user is prompted to enter their email address. 
- [x] After sending a POST request with required email, server compiles a e-mail with a generated hyperlink attached to it.
- [x] The generated hyperlink leads to the form where the user can enter a new password
- [x] After a successful POST request (where the password matches) redirect the user to the login page.
-----------------
Prerequisites:
- [x] Kendo UI added
- [x] Mail service implementation
- [x] Dynamic hyperlink generator/controller/something (?)

In other words, these implementations are required:
- [x] Reset password page (email form)
- [x] Reset password page 2 (password form)
- [x] Login page with password reset option
